packages/@uppy/core/package.json

```diff
@@ -28,6 +28,17 @@
     "type": "git",
     "url": "git+https://github.com/transloadit/uppy.git"
   },
+  "exports": {
+    ".": "./lib/index.js",
+    "./src/style.scss": "./src/style.scss",
+    "./dist/style.min.css": "./dist/style.min.css",
+    "./dist/style.css": "./dist/style.css",
+    "./lib/Uppy.js": "./lib/Uppy.js",
+    "./lib/EventManager.js": "./lib/EventManager.js",
+    "./lib/UIPlugin.js": "./lib/UIPlugin.js",
+    "./lib/Restricter.js": "./lib/Restricter.js",
```

Comment:
Why are these separate? If some imports starting failing, shouldn't we move it to `index.js`?